src/components-examples/material/table/index.ts

```diff
@@ -46,39 +46,47 @@ import {TableHarnessExample} from './table-harness/table-harness-example';
 import {TableWithRipplesExample} from './table-with-ripples/table-with-ripples-example';
 import {TableColumnStylingExample} from './table-column-styling/table-column-styling-example';
 import {TableRowBindingExample} from './table-row-binding/table-row-binding-example';
+import {
+  TableDynamicArrayDataExample
+} from './table-dynamic-array-data/table-dynamic-array-data-example';
+import {
+  TableDynamicObservableDataExample
+} from './table-dynamic-observable-data/table-dynamic-observable-data-example';
 
 export {
-  TableBasicExample,          TableFlexBasicExample,
-  TableDynamicColumnsExample, TableExpandableRowsExample,
-  TableFilteringExample,      TableFooterRowExample,
-  TableHttpExample,           TableMultipleHeaderFooterExample,
-  TableOverviewExample,       TablePaginationExample,
-  TableRowContextExample,     TableSelectionExample,
-  TableSortingExample,        TableStickyColumnsExample,
-  TableStickyComplexExample,  TableStickyComplexFlexExample,
-  TableStickyFooterExample,   TableStickyHeaderExample,
-  TableTextColumnExample,     TableTextColumnAdvancedExample,
-  TableWrappedExample,        WrapperTable,
-  TableReorderableExample,    TableRecycleRowsExample,
-  TableHarnessExample,        TableColumnStylingExample,
-  TableRowBindingExample,     TableWithRipplesExample,
+  TableBasicExample,             TableFlexBasicExample,
+  TableDynamicColumnsExample,    TableExpandableRowsExample,
+  TableFilteringExample,         TableFooterRowExample,
+  TableHttpExample,              TableMultipleHeaderFooterExample,
+  TableOverviewExample,          TablePaginationExample,
+  TableRowContextExample,        TableSelectionExample,
+  TableSortingExample,           TableStickyColumnsExample,
+  TableStickyComplexExample,     TableStickyComplexFlexExample,
+  TableStickyFooterExample,      TableStickyHeaderExample,
+  TableTextColumnExample,        TableTextColumnAdvancedExample,
+  TableWrappedExample,           WrapperTable,
+  TableReorderableExample,       TableRecycleRowsExample,
+  TableHarnessExample,           TableColumnStylingExample,
+  TableRowBindingExample,        TableWithRipplesExample,
+  TableDynamicArrayDataExample,  TableDynamicObservableDataExample,
```

Comment:
nit: IMO we shouldn't be stacking the examples in columns like this, because it's prone to merge conflicts.

Reply:
Yeah I'm surprised it was like this. It was a pain to set up. I'll switch it to a normal list